NitroxClient/Communication/Packets/Processors/SimulationOwnershipChangeProcessor.cs

```diff
@@ -61,7 +61,7 @@ private void StartBroadcastingEntityPosition(NitroxId id)
             {
                 EntityPositionBroadcaster.WatchEntity(id, gameObject.Value);
             }
-#if DEBUG
+#if DEBUG && ENTITY_LOG
```

Comment:
Okay, imo then might be worth actually adding this in as a build configuration into the project file so we can just switch within vs rather than having to go in ourselves as add it each time

Reply:
The goal is to disable the logging. I know this is typically no good behavior but in this case with it spamming a lot of error messages which have no value (and are disabled on release) it's ok imo.